src/trans/include/ectrans/gpnorm_transtl.h

```diff
@@ -0,0 +1,63 @@
+! (C) Copyright 2000- ECMWF.
+! (C) Copyright 2013- Meteo-France.
+! 
+! This software is licensed under the terms of the Apache Licence Version 2.0
+! which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+! In applying this licence, ECMWF does not waive the privileges and immunities
+! granted to it by virtue of its status as an intergovernmental organisation
+! nor does it submit to any jurisdiction.
+!
+
+INTERFACE
+SUBROUTINE GPNORM_TRANSTL(PGP,KFIELDS,KPROMA,PAVE,KRESOL)
+
+
+!**** *GPNORM_TRANSTL* - calculate grid-point norms
+
+!     Purpose.
+!     --------
+!        calculate grid-point norms using a 2 stage (NPRTRV,NPRTRW) communication rather
+!        than an approach using a more expensive global gather collective communication
+
+!**   Interface.
+!     ----------
+!     CALL GPNORM_TRANSTL(...)
+
+!     Explicit arguments :
+!     --------------------
+!     PGP(:,:,:) - gridpoint fields (input)
+!                  PGP is  dimensioned (NPROMA,KFIELDS,NGPBLKS) where
+!                  NPROMA is the blocking factor, KFIELDS the total number
+!                  of fields and NGPBLKS the number of NPROMA blocks.
+!     KFIELDS     - number of fields (input)
+!                   (these do not have to be just levels)
+!     KPROMA      - required blocking factor (input)
+!     PAVE        - average (output)
+!     KRESOL      -  resolution tag (optional)
+!                    default assumes first defined resolution
+!
+
+!     Author.
+!     -------
+!        George Mozdzynski *ECMWF*
```

Comment:
More details ... should author and date not be consistent with that in the routine itself?

Reply:
Have updated gpnorm_transtl.h to match gpnorm_transtl.F90.